### PR TITLE
adds progress circles to experienced player home page

### PR DESCRIPTION
**Pull request by agarg5**
- Adds Progress Circles to experienced player home page. 
- Adds Countdown to experienced player home page. 
- Makes long titles fit without text going out of bounds on all screen sizes.

Progress Circles:

Countdown:

@sharife please let me know if the way it looks should be different on mobile

### CODE REVIEW

**Comment by agarg5** [comment]
TODO improve styles:
current large:
image
current small:
image

@akrolsmir if you have the time, it would be great if you could improve the styling

File: src/views/landing/ExperiencedPlayerHome/components/BannerSlide.vue [hunk 1]
```diff
@@ -33,17 +36,65 @@
...
+        name: 'progress-circle:designs-submissions',
+        progress: this.designs_to_be_synthesized,
+        total: this.max_designs,
+      },
+      {
+        name: 'progress-circle:my-submissions',
+        progress: this.total_submitted_solutions_of_user,
+        total: this.total_submitted_solutions,
+      },
+    ];
```
[the remark this answers is not in the document]
Comment (agarg5) [reply src/views/landing/ExperiencedPlayerHome/components/BannerSlide.vue]: @luxaritas thanks made these changes for the first two bullets. I will ask Tamas for an example for what a not null example for the `project_closes` field looks like to make the countdown timer and also to add the data to the endpoints so this can show in the lab

[the remark this answers is not in the document]
Comment (agarg5) [reply src/views/landing/ExperiencedPlayerHome/components/BannerSlide.vue]: added countdown using timestamp
image

**Comment by luxaritas** [comment]
I have my .env.local set to use prod, and it appears all slides are using the same timestamp. One of them (RNA backbone stability) shouldn't have the countdown at all (just both circles). All three should also have at least one circle (the one designating user designs / max designs per user). The fields were not all set properly on the server, but I just fixed that

**Comment by luxaritas** [comment]
Also, I think the sizing/positioning of the countdown timer could be improved - @sharife could you comment on that?

**Comment by agarg5** [comment]
@sharife this is what the countdown looks like on mobile: 
image

Please let me know if this is OK or if we need to change anything about the styling.

**Comment by luxaritas** [comment]
FWIW, my concern was with desktop rather than mobile

File: src/components/Sidebar/CalendarPanel.vue [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template>
   <SidebarPanel :isInSidebar="isInSidebar" header="calendar" headerIcon="@/assets/calendar.svg">
-    <DatePicker mode="range" v-model="dates" color="yellow" is-dark is-inline />
+    <vc-date-picker mode="range" v-model="dates" color="yellow" is-dark is-inline />
```
Comment (agarg5) [inline src/components/Sidebar/CalendarPanel.vue:3]: unrelated change to avoid error/warning
